packages/elements/list/src/getListDeleteForward.ts

```diff
@@ -22,15 +30,52 @@ export const getListDeleteForward = (editor: SPEditor) => {
   const res = getListItemEntry(editor, {});
 
   let moved: boolean | undefined = false;
-  if (!isSelectionAtBlockEnd(editor) || !res) {
+  if (!isSelectionAtBlockEnd(editor)) {
     return moved;
   }
 
+  if (!res) {
+    const afterPoint = Editor.after(editor, editor.selection!.focus.path);
```

Comment:
`pointAfterSelection` maybe? :thinking:

Reply:
Good idea will make a commit tomorrow for theses